### DISCUSSION TITLE
Upgrade to 7.31 from 7.20 breaks application

### DISCUSSION THREAD

**Discussion by MilesGibson** [Blazored/Modal#569] [General]
So I noticed this morning when checking NuGet packages that there is an upgrade to 7.31.  Great!  I have been using version 7.20 without issue for a long time.

HOWEVER: after I upgraded and did a ReBuild All, suddenly Blazored.Modal was no longer available to reference!  I finally downgraded back to 7.20 and eventually it all compiled and works again.  Whew!

Why does this happen?  Is there a NuGet cache or something I need to clear before upgrading to the latest version?

TIA,

Miles

**Comment by chrissainty** [comment]
This has been raised in #566. I can't replicate this issue so far. If you can provide a repro it would be great if you could add a comment to the issue.

**Comment by MilesGibson** [comment]
[TEST7.zip](https://github.com/Blazored/Modal/files/14607145/TEST7.zip)
See attached a test blazor .net7 application (with DevExpress references).  Does not recognize Blazored.Modal namespace.

Cheers,

Miles

**Reply by chrissainty** [reply]
Thank Miles, but could you upload this to a public GitHub repository. In the nicest possible way, downloading and extracting zip files from random people on the internet isn't a great idea 😉

**Reply by MilesGibson** [reply]
Sorry you don't have adequate anti-virus protection.  I will see what I can do to get it up to a different Github account than this one.

**Reply by MilesGibson** [reply]
https://github.com/MilesGibson/TEST7

Hope this is acceptable to  your computer, in the nicest possible way from one random person to another.

**Reply by chrissainty** [reply]
Yeah I think you should find another library to use fella.
